arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/continuations/Effect.kt

```diff
@@ -568,7 +568,7 @@ import kotlin.coroutines.resume
  * ```
  * <!--- KNIT example-effect-guide-13.kt -->
  */
-public interface Effect<R, A> {
+public interface Effect<out R, A> {
```

Comment:
We should probably make `A` also `out`.

Reply:
That way it will behave the same as `Either` does today:

```kotlin
val either: Either<String, Int> = 1.right()
val widen: Either<CharSequence, Number> = either
```

Reply:
Will do ☺️👍🏾